osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -67,7 +67,9 @@ def on_moved(self, event):
         item = self._get_item_by_path(src_path)
 
         if isinstance(item, Node):
-            AlertHandler.warn('Cannot manipulate components locally. {} will stop syncing'.format(item.title))
+            if item.title != dest_path.name:
+                AlertHandler.warn('Cannot manipulate components locally. {} will stop syncing'.format(item.title))
+                # os.rename(str(dest_path), item.path)
```

Comment:
Remove this line.

(And for future reference: If you were moving files/folders- in a program that had far more control over the user's behavior than osf offline does- you'd probably want to look into something like `shutil.move` instead)